src/components/Editor/Invitees/InviteesList.vue

```diff
@@ -214,10 +230,36 @@ export default {
 		hasOrganizer() {
 			return this.calendarObjectInstance.organizer !== null
 		},
-
 		organizerDisplayName() {
 			return organizerDisplayName(this.calendarObjectInstance.organizer)
 		},
+		organizerSelection() {
+			const organizers = []
+			const owner = this.principalsStore.getPrincipalByUrl(this.calendarInstance.owner)
+			const principal = this.principalsStore.getCurrentUserPrincipal
+			if (owner) {
+				organizers.push({
+					id: owner.id,
+					label: owner.displayname,
+					address: owner.emailAddress
+				})
+			}
+			if (principal && owner.id !== principal.id) {
+				organizers.push({
+					id: principal.id,
+					label: principal.displayname,
+					address: principal.emailAddress
+				})
+			}
+
+			return organizers
+		},
+		organizerSelected() {
+			return {
+				label: this.calendarObjectInstance.organizer.commonName,
```

Comment:
This line and others break when `this.calendarObjectInstance.organizer` is `null`

Reply:
Fixed